### DISCUSSION TITLE
My box is X96 max 4G+64G,I cant use wifi,please help.

### DISCUSSION THREAD

**Discussion by The4thWall** [devmfc/debian-on-amlogic#69] [Q&A]
I used ./wifi-connect.sh, it prompt Network is down(-100),and no wifi SSID list,then prompt Operation not permitted
run ifconfig wlan0 up, it prompt SIOCSIFFLAGS: Operation not permitted,How can I do?
thx!
WIFI:Fn-Link 6222B-SRB（RTL8822BS-CG）

**Comment by NickelNuts** [comment]
"S905W2 boxes: WiFi does not work."  Maybe a USB wifi adapter if you have one?

**Comment by NickelNuts** [comment]
On 2nd thought, it looks like the firmware is loaded.  I think you have to be root to run ifconfig.  So, run using "sudo ./wifi-connect.sh" if this is from a user account.  If that fails, install sudo with "apt install sudo"  then add your user to the netdev and sudo groups

**Reply by The4thWall** [reply]
THX，I failed to use  a root account ，and I failed to use sudo too...
